app/jobs/resolution_proofing_job.rb

```diff
@@ -46,6 +46,8 @@ def perform(
       timer: timer,
       applicant_pii: applicant_pii,
       should_proof_state_id: should_proof_state_id,
+      capture_secondary_id_enabled:
```

Comment:
just checking in here, so we chose to augment the existing job, instead of duplicate to a second job for double address verification?